Library/Formula/tokyo-tyrant.rb

```diff
@@ -15,6 +15,11 @@ class TokyoTyrant < Formula
     patch :DATA
   end
 
+  patch :p0 do
+    url 'https://gist.githubusercontent.com/actsasflinn/348633/raw/32561ebf6487650ce86d0184f0ca35bb8110d04c/macosx_snow_leopard_socket_fix.patch'
+    sha1 '2b73fe52c37cd41b8e13112c0bc083bce8ae3c9e'
+  end
```

Comment:
Emailed them and created [another patch on Gist](https://gist.github.com/bow-fujita/a338e35b95605ad63147) to explain the issue.